src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -321,6 +321,13 @@ private static string GetFixCategory(DiagnosticSeverity severity)
                     return SpecializedTasks.EmptyImmutableArray<UnifiedSuggestedActionSet>();
                 }
 
+                // 'CodeActionRequestPriority.Low' is reserved for suppression/configuration code fixes.
+                // No code refactoring should have this request priority.
```

Comment:
i would prefer this not be true.  as an example we have things like the 'wrapping' fixes which are always low priority.

Reply:
I introduced `CodeActionRequestPriority.Lowest` and now that one is dedicated for suppression/configuration fixes.